src/Components/AddBook/AddBook.js

```diff
@@ -1,22 +1,75 @@
-import React from 'react';
+import uuid from 'react-uuid';
+import { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { addBook, deleteBook } from '../../redux/books/books';
 import './AddBook.css';
 
-const AddBook = () => (
-  <div>
-    <div className="container" id="addRow">
+const AddBook = () => {
+  const dispatch = useDispatch();
+  const bookList = useSelector((state) => state.books.value);
+
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  return (
+    <div className="container py-4">
       <div className="row">
-        <hr className="col-12 my-5" />
-        <h2 className="col-12 addTitle my-3"> ADD NEW BOOK </h2>
+        <div>
+          {bookList.map((book) => (
+            <div key={book}>
+              <div className="row">
+                <h3 className="col-12 bookTitle">{book.title}</h3>
+                <h3 className="col-12 author">{book.author}</h3>
+              </div>
+              <button
+                type="button"
+                className="col-3 my-2 removeButton"
+                onClick={() => {
+                  dispatch(deleteBook({ id: book.id }));
+                }}
+              >
+                Remove
```

Comment:
- [ ] Based on the requirements of this project, this button should be found in another component called `Book`(We need to learn how to make use of several components in our app :+1: ). This button should use a function `removeBookFromStore` (name can be different) made in the component(We needs to learn how to craft a function and make use of it :+1: ). Kindly fix :wrench: this

*